packages/sdk/app-framework/src/vite-plugin/boot-loader/loader-app/boot-loader.css

```
@@ -143,94 +147,56 @@ body {
-  mask: radial-gradient(
-    circle closest-side,
-    transparent calc(100% - var(--boot-loader-line-thickness) - var(--boot-loader-line-feather)),
-    #000 calc(100% - var(--boot-loader-line-thickness))
+  --boot-loader-arc: 0;
+  mask: conic-gradient(
+    from 0deg at 50% 50%,
+    transparent 0,
+    transparent calc((100 - var(--boot-loader-arc)) * 1%),
+    #000 calc((100 - var(--boot-loader-arc)) * 1%),
+    transparent 100%
   );
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)